.github/workflows/update-insecure-dependencies.yaml

```
@@ -65,9 +67,9 @@ jobs:
             After update:
             ${{ env.SCAN_OUTPUT_AFTER }}
           delete-branch: true
-          title: "chore(deps): security update"
+          title: "chore(deps): security update for ${{ matrix.branch }}"
```

```suggestion
          title: "chore(deps): security update
```

The labels are displayed in the PR list so no need to make this redundant